src/sas/qtgui/Plotting/LinearFit.py

```diff
@@ -304,32 +308,36 @@ def floatInvTransform(self, x):
         panel.
 
         """
-        # TODO: refactor this. This is just a hack to make the
-        # functionality work without rewritting the whole code
-        # with good design (which really should be done...).
-        if self.xLabel == "x":
-            return x
-        elif self.xLabel == "x^(2)":
-            return numpy.sqrt(x)
-        elif self.xLabel == "x^(4)":
-            return numpy.sqrt(numpy.sqrt(x))
-        elif self.xLabel == "log10(x)":
-            return numpy.power(10.0, x)
-        elif self.xLabel == "ln(x)":
-            return numpy.exp(x)
-        elif self.xLabel == "log10(x^(4))":
-            return numpy.sqrt(numpy.sqrt(numpy.power(10.0, x)))
-        return x
+        from sas.qtgui.Utilities.GuiUtils import xyTransform
```

Comment:
is this import so expensive that you had to defer it? If not, please consider moving it to the top.